Lib/test/test_pdb.py

```diff
@@ -2350,6 +2350,30 @@ def test_pdb_ambiguous_statements():
     (Pdb) continue
     """
 
+def test_pdb_f_trace_lines():
+    """GH-80675
+
+    pdb should work even if f_trace_lines is set to False on some frames.
+
+    >>> reset_Breakpoint()
+
+    >>> def test_function():
+    ...     import sys
+    ...     frame = sys._getframe()
+    ...     frame.f_trace_lines = False
+    ...     import pdb; pdb.Pdb(nosigint=True, readrc=False).set_trace()
+    ...     if frame.f_trace_lines != False:
+    ...         print("f_trace_lines is not reset after continue!")
```

Comment:
Not sure I understand this test. Why should continue delete the breakpoints?

Reply:
It has nothing to do with the breakpoints. `reset_Breakpoint()` is used to clear all the breakpoints - to make sure there's no breakpoint left for pdb - otherwise `f_trace_lines` will NOT be reverted to what it used to be.